packages/orchestration/test/staking-ops.test.ts

```diff
@@ -8,21 +8,21 @@ import {
   MsgDelegateResponse,
   MsgUndelegateResponse,
 } from '@agoric/cosmic-proto/cosmos/staking/v1beta1/tx.js';
-import { makeScalarBigMapStore } from '@agoric/vat-data';
+import { makeScalarBigMapStore, type Baggage } from '@agoric/vat-data';
```

Comment:
Any particular reason to convert this to .ts?

Moving code between tests and non-tests is something I do pretty often, and having them in different languages adds friction.

Reply:
I wanted to write the new test in `.ts` and I thought it would be better for all tests in the package to be consistent, for moving code between them. I didn't consider moving code between tests and non-tests often. That would be a reason to keep tests in `.js`, to weigh against the benefits of using `.ts` syntax.

The reason I think tests merit `.ts` is that they often deal with mock values and it's relatively tedious to cast those in `.js`.

What are some examples of copying between tests and non-tests?

Reply:
> better for all tests in the package to be consistent

that makes sense

> What are some examples of copying between tests and non-tests?

some dapp-agoric-basics end-to-end client stuff went back and forth between `test/` `src/` and `tools/` a little while ago.
IOU some more examples.